src/t8_cmesh/t8_cmesh_vertex_connectivity/t8_cmesh_vertex_conn_vertex_to_tree.cxx

```diff
@@ -113,8 +113,7 @@ t8_cmesh_vertex_conn_vertex_to_tree::add_vertex_to_tree ([[maybe_unused]] const
  *    (\a local_tree_id_a == \a local_tree_id_b and \a local_tree_vertex_a < \a local_tree_vertex_b)
  */
 static inline bool
-t8_cmesh_tree_vertex_pair_compare (t8_cmesh_vertex_conn_vertex_to_tree::tree_vertex_pair const& pair_a,
-                                   t8_cmesh_vertex_conn_vertex_to_tree::tree_vertex_pair const& pair_b)
+t8_cmesh_tree_vertex_pair_compare (tree_vertex_pair const& pair_a, tree_vertex_pair const& pair_b)
```

Comment:
Please document this function.

Reply:
I am not sure what comments i could add.
Could you please specify?

Reply:
Ah sorry, my bad. I meant the documentation of the input parameters, but you already did this. I did not properly see it in GH.